ExpressApp.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const path = require('path');
+const _ = require('lodash');
+const moment = require('moment');
+const yaml = require('js-yaml');
+const express = require('express');
+const morgan = require('morgan');
+const bodyParser = require('body-parser');
+const lib = require('./lib');
+const config = lib.config;
+const logger = lib.logger;
+const middleware = lib.middleware;
+const connectTimeout = require('connect-timeout');
+
+class ExpressApp {
```

Comment:
Use name corresponding to our use case ? like `ReportApp` ??
And rename across the file.